test/unit/Subscription.test.js

```diff
@@ -19,6 +19,16 @@ const createMsg = (
 
 const msg = createMsg()
 
+const assertRejects = async (promise) => {
+    let thrown
+    try {
+        await promise
+    } catch (err) {
+        thrown = err
+    }
+    assert(thrown != null)
+}
```

Comment:
Since we're using jest, you can use this form: `await expect(promise).rejects.toBeTruthy()`